### DISCUSSION TITLE
Welcome to angular-learn Discussions!

### DISCUSSION THREAD

**Discussion by kevin921115** [kevin921115/angular-learn#13] [Q&A] [answered]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kevin921115** [comment, marked as answer]
hi

**Reply by kevin921115** [reply]
hi

**Comment by prettyTiger1022** [comment]
hi

**Reply by kevin921115** [reply]
hi nice to meet you here
